src/tactic/tidy.lean

```diff
@@ -79,6 +79,8 @@ meta def tidy (cfg : tidy.cfg := {}) := tactic.tidy.core cfg >> skip
 namespace interactive
 open lean.parser interactive
 
+/-- Use a variety of conservative tactics to solve goals.
+`tidy?` reports back the tactic script it found. -/
```

Comment:
Let's add the rest of the info from the tactic docs:
```suggestion
`tidy?` reports back the tactic script it found.
The default list of tactics is stored in `default_tidy_tactics`.
This list can be overridden using `tidy { tactics := ... }`.
(The list must be a `list` of `tactic string`, so that `tidy?`
can report a usable tactic script.) -/
```
BTW Mergify is waiting for travis, who will never report back because [skip ci] is in the commit name. It looks like it should be possible to configure mergify to merge such PRs directly without waiting for travis.